### DISCUSSION TITLE
How could I convert a JavaScript object to a JSON object?

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#9440] [Q&A]
Sorry guys for the stupid question

I try it and does not works:

![image](https://user-images.githubusercontent.com/32253743/134797443-a922f66e-833d-4334-93a3-3f14e1e06a50.png)

However, If I use JSON.stringify + JSON.parse works:

![image](https://user-images.githubusercontent.com/32253743/134797462-063b4c0f-fa54-4ed4-ab5d-5ee218dca5a4.png)

Is there any better way to do it? Use JSON.stringify + JSON.parse in sequence is SOOOOOOOOOO weird

**Comment by ryands17** [comment]
@felinto-dev 👋 

This is just a TypeScript thing, so you could just add this to make the warning go away:

```ts
rawTransactionJson: transaction as any
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
